__tests__/app.test.js

```diff
@@ -44,6 +44,58 @@ describe('GET /api/topics', () => {
       });
   });
 });
+describe('GET /api/articles', () => {
+  test('returned array is not empty', () => {
+    return request(app)
+      .get('/api/articles')
+      .expect(200)
+      .then((response) => {
+        expect(response.body.articles.length).not.toBe(0);
+      });
```

Comment:
This test is possibly covered by the second test, if your API returns an empty array the `forEach` in the next test won't run. It might be more impactful if it instead asserts that the array should be the specific length you expect it to be _(based on the test data which will always be used for these tests)_